boards/xg23b-pk6068a/Makefile.features

```diff
@@ -0,0 +1,6 @@
+CPU = efm32
+CPU_FAM = efr32zg23
+CPU_MODEL = efr32zg23a020f512gm48
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_uart
```

Comment:
Isn't it necessary for this board to include the makefiles from `boqards/common/silabs`?